Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1064,6 +1064,16 @@ ScrollbarWidth RenderLayerScrollableArea::scrollbarWidthStyle()  const
     return ScrollbarWidth::Auto;
 }
 
+std::optional<ScrollbarColor> RenderLayerScrollableArea::scrollbarColorStyle() const
+{
+    if (m_layer.renderBox()) {
+        if (auto value = m_layer.renderer().style().scrollbarColor().tryValue())
+            return ScrollbarColor { m_layer.renderer().style().colorResolvingCurrentColor(value->thumb), m_layer.renderer().style().colorResolvingCurrentColor(value->track) };
```

Comment:
```suggestion
            return {
                .thumb = m_layer.renderer().style().colorResolvingCurrentColor(value->thumb),
                .track = m_layer.renderer().style().colorResolvingCurrentColor(value->track)
            };
```
(Maybe ScrollbarColor is necessary here because of the optional, not sure)

Reply:
Fixed.